parser_library/src/context/instruction_details.thh

```diff
@@ -1097,7 +1097,7 @@ DEFINE_INSTRUCTION(PALB, RRE_0, 1209, UNI_ESA_SINCE_ZOP, "Purge ALB", privileged
 DEFINE_INSTRUCTION(PC, S_1_u, 1183, UNI_ESA_XA_370_SINCE_ZOP, "Program Call", privileged_conditionally, branch_argument_unknown)
 DEFINE_INSTRUCTION(PCC, RRE_0, 872, UNI_SINCE_Z11, "Perform Cryptographic Computation", pcc_special)
 DEFINE_INSTRUCTION(PCKMO, RRE_0, 1162, UNI_SINCE_Z10, "Perform Crypto. Key Mgmt. Operations", privileged, has_parameter_list)
-DEFINE_INSTRUCTION(PFCR, RSY_a_3, 0, UNI_SINCE_Z17, "Perform functions with concurrent results")
+// DEFINE_INSTRUCTION(PFCR, RSY_a_3, 0, UNI_SINCE_Z17, "Perform functions with concurrent results") // removed before GA
```

Comment:
Just to be sure - is this still supposed to be commented out?

Reply:
Yes, the instruction was removed from the doc and seems to be scheduled for removal from hlasm by a ptf.
But I want to keep it as it looks like it is actually implemented in HW.